docs/DeveloperGuide.md

```diff
@@ -29,9 +29,14 @@ The rest of the App consists of these components.
 
 ### TaskList component
 
+![TaskListStructure](https://raw.githubusercontent.com/wish2023/tp/DG-tasklist/docs/structures/TaskListStructure.png)
+
 **API** :
 [`TaskList.java`](https://github.com/AY2021S1-CS2113T-W12-2/tp/blob/master/src/main/java/athena/TaskList.java)
 
+1. The `TaskList` stores task data in `Task` type objects.
+2. The `TaskList` is updated in `Athena`.
```

Comment:
After #131 it will be in `LogicManager`.

Reply:
Yup let's change it to `logicManager` once that's merged